code/modules/antagonists/wizard/arbiter.dm

```diff
@@ -79,3 +79,78 @@
 	W.registered_name = H.real_name
 	W.update_label()
 	..()
+
+//Spawner
+/obj/effect/mob_spawn/human/arbiter
+	name = "The Arbiter"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper_s"
+	short_desc = "You are The Arbiter."
+	important_info = "You are hostile to L-Corp. Assist abnormalities in killing them all."
+	outfit = /datum/outfit/arbiter
+	max_integrity = 9999999
+	density = TRUE
+	roundstart = FALSE
+	death = FALSE
+
+/obj/effect/mob_spawn/human/arbiter/special(mob/living/new_spawn)
+	new_spawn.mind.add_antag_datum(/datum/antagonist/wizard/arbiter)
+
+/obj/effect/mob_spawn/human/arbiter/complete
+
+/obj/effect/mob_spawn/human/arbiter/complete/special(mob/living/new_spawn)
+	new_spawn.mind.add_antag_datum(/datum/antagonist/wizard/arbiter/complete)
+
+/datum/antagonist/wizard/arbiter/complete
+	name = "Arbiter"
+	spell_types = list(
+		/obj/effect/proc_holder/spell/aimed/fairy,
+		/obj/effect/proc_holder/spell/aimed/pillar,
+		/obj/effect/proc_holder/spell/aoe_turf/repulse/arbiter,
+		/obj/effect/proc_holder/spell/aoe_turf/knock/arbiter,
+		/obj/effect/proc_holder/spell/aoe_turf/singularity
```

Comment:
minor nitpick: all of the lists contents should have a comma at the end, even the ending ones
this makes the list look better when being edited in the future on github's lines changed thingy